yaml/src/main/yaml/PostgresToIceberg.yaml

```diff
@@ -0,0 +1,55 @@
+template:
+  name: "Postgres_To_Iceberg_Yaml"
+  display_name: "Postgres to Iceberg (Yaml)"
+  description: "A template for Postgres to Iceberg."
+  flex_container_name: "postgres-to-iceberg-yaml"
+  category: "BATCH"
+
+  options_file:
+    - "jdbc_options"
```

Comment:
Given that we have separate schema-transforms for Databases (MySQL, Postgre etc.) should we also have separate options files ? This will also better translate to managed replacement when we do it.